modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/form_builder_rule_builder.js

```diff
@@ -522,6 +523,22 @@ AUI.add(
 						A.one('.lfr-ddm-add-rule').addClass('hide');
 					},
 
+					_handleSaveRuleDraft: function(event) {
+						var instance = this;
+
+						var rule = {};
+
+						if (!event.actions && !event.conditions && event['logical-operator'] != '') {
```

Comment:
can you assign `event.actions`, `event.conditions` and `event['logical-operator']` into variables and reuse them inside the condition?